src/windows/scanner.cc

```diff
@@ -186,8 +186,20 @@ ScanMountpoints(std::vector<drivelist::mountpoint_s> *const output) {
     ULONG number;
     drivelist::Debug("Getting device number");
     result = drivelist::volume::GetDeviceNumber(volume, &number);
-    if (FAILED(result))
+    if (FAILED(result)) {
+
+      // This error, "Invalid Function", can happen when attempting
+      // to get the device number of a virtual disk.
```

Comment:
IMHO you should get rid of the "Invalid Function" part and just change it to `This error can happen when attempting to get the device number of a virtual disk.` with a similar change also made to the commit message.